### PR TITLE
make clearer that the paid plans don't include community

### PR DESCRIPTION
**Pull request by hugovalente-pm**

### CODE REVIEW

**Comment by andrewm4894** [comment]
@hugovalente-pm shall we merge this?

**Review by andrewm4894 — APPROVED** [review]

File: blog/2023-02-10-introducing-netdata-paid-plans/index.md [hunk 1]
```diff
@@ -18,7 +18,7 @@ We are excited to announce that we are introducing new paid subscriptions to our
...
 
-The new paid subscriptions are designed to provide additional features and capabilities for businesses that need tighter and customizable integration of the free monitoring solution to their processes, and they will be divided into three different plans: COMMUNITY, PRO, and BUSINESS. Each plan will offer a different set of features and capabilities to meet the needs of businesses of different sizes and with different monitoring requirements.
+The new paid subscriptions are designed to provide additional features and capabilities for businesses that need tighter and customizable integration of the free monitoring solution to their processes, these. Together with the free plan, the plans will be divided into three different plans: COMMUNITY, PRO, and BUSINESS. Each plan will offer a different set of features and capabilities to meet the needs of businesses of different sizes and with different monitoring requirements.
```
[the remark this answers is not in the document]
Comment (andrewm4894) [reply blog/2023-02-10-introducing-netdata-paid-plans/index.md:21]: small typo?

**Comment by andrewm4894** [comment]
@hugovalente-pm i messed this up - we would need to just to a fresh PR instead - apologies.

**Comment by hugovalente-pm** [comment]
seems it is fixed, no @andrewm4894 ?

**Comment by andrewm4894** [comment]
i think was cleaned up here actually. https://github.com/netdata/blog/pull/207/files

anyways feel free to make any additional changes, just that i merged this PR by mistake and then reverted it. But i think that all maybe superseeded by #207 anyway. OR if not just to let you know so can make fresh PR if needed.

**Pull request merged**